sphinxcontrib/needs/directives/needfilter.py

```diff
@@ -114,7 +114,7 @@ def process_needfilters(app, doctree, fromdocname):
                 continue
 
             plantuml_block_text = ".. plantuml::\n" "\n" "   @startuml" "   @enduml"
-            puml_node = plantuml(plantuml_block_text, **dict())
+            puml_node = plantuml(plantuml_block_text, **{})
```

Comment:
what is the `**dict()` arg doing anyway? can this just be removed entirely?

Reply:
i've removed them. let's see if the CI is happy

Reply:
not sure,  but often a node gets 2 arguments. So I think the plantuml-node needs it.

Reply:
i'm just trying to figure out what this expands to. the `**` operator turns a dictionary into keyword arguments, but since it's an empty dictionary, surely that results in 0 keyword arguments, which is equivalent to calling the method with only a single arg.

right?

i've not seen this construction before, so it's got me second-guessing myself

Reply:
ohh yes, for sure you are right. So it makes no sense.

Reply:
the 'linkcheck' job has passed this time, so it appears to have been some transient error